website/thaliawebsite/settings.py

```diff
@@ -944,3 +948,23 @@ def from_env(
         "auth",
     ],
 }
+MONEYBIRD_ADMINISTRATION_ID = os.environ.get("MONEYBIRD_ADMINISTRATION_ID", None)
+MONEYBIRD_API_KEY = os.environ.get("MONEYBIRD_API_KEY", None)
+
+MONEYBIRD_SYNC_ENABLED = MONEYBIRD_ADMINISTRATION_ID and MONEYBIRD_API_KEY
+
+MONEYBIRD_CUSTOM_FIELD_ID = os.environ.get("MONEYBIRD_CUSTOM_FIELD_ID", None)
```

Comment:
What is the purpose of this custom field (what do we put in the custom field? is it the member id in concrexit? in that case I would consider refactoring to MONEYBIRD_MEMBER_PK_CUSTOM_FIELD_ID or something)

Reply:
Yes it is the member id/pk